totalRP3/Modules/Register/Characters/RegisterRelations.lua

```diff
@@ -0,0 +1,348 @@
+-- Copyright The Total RP 3 Authors
+-- SPDX-License-Identifier: Apache-2.0
+
+local Globals = TRP3_API.globals;
+local loc = TRP3_API.loc;
+local EMPTY = Globals.empty;
+local tcopy = TRP3_API.utils.table.copy;
+local get = TRP3_API.profile.getData;
+local getPlayerCurrentProfile = TRP3_API.profile.getPlayerCurrentProfile;
+local displayDropDown = TRP3_API.ui.listbox.displayDropDown;
+local getProfiles = TRP3_API.profile.getProfiles;
+
+-- These functions get replaced by the proper TRP3 ones once the addon has finished loading
+local function getPlayerCompleteName()
+	return TRP3_API.globals.player
+end
+local function getCompleteName()
+	return UNKNOWN
+end
+
+TRP3_API.register.relation = {};
+
+
+TRP3_API.RegisterCallback(TRP3_Addon, TRP3_Addon.Events.WORKFLOW_ON_LOAD, function()
+	getCompleteName, getPlayerCompleteName = TRP3_API.register.getCompleteName, TRP3_API.register.getPlayerCompleteName;
+end);
```

Comment:
If there's no circumstances where these two can be called ahead of the addon finishing loading, I'd suggest just obliterating them and making the calls to getCompleteName and getPlayerCompleteName fully qualified at the point of the calls.